tf2/include/tf2/LinearMath/MinMax.h

```diff
@@ -18,25 +18,25 @@ subject to the following restrictions:
 #define GEN_MINMAX_H
 
 template <class T>
-TF2SIMD_FORCE_INLINE const T& tf2Min(const T& a, const T& b) 
+TF2_PUBLIC TF2SIMD_FORCE_INLINE const T& tf2Min(const T& a, const T& b) 
```

Comment:
Ah, that makes sense. Clearly I'm in unfamiliar territory here. So I should remove that declaration from all the template functions? Anywhere else?